proto/api/v1/metadata_service.proto

```diff
@@ -63,11 +63,35 @@ message DatabaseStatus {
     string database_version = 3;
 }
 
-
 message DatabaseBackupStatus {
     storage.BackupInfo backup_info = 1;
 }
 
+// Provides availability of certain functionality of Central Services in the current configuration.
+// The initial intended use is to disable certain functionality that does not make sense in the Cloud Service context.
+message CentralServicesCapabilities {
+    enum CapabilityStatus {
+        // "Unknown" means "not disabled".
+        // The reason it's not called "Enabled" is that extended checks might be required to confirm true ability to use
+        // e.g. container AWS IAM role or GCP workload identity, and we can't positively say "Enabled" while such checks
+        // aren't implemented.
+        // This means the user should be allowed to use the capability (both via UI and API) but an attempt may not be
+        // successful if the corresponding configuration does not match the actual environment.
+        Unknown = 0;
+
+        // Capability is disabled, meaning the corresponding UI should be disabled and attempts to use related APIs
+        // should lead to errors.
+        Disabled = 1;
+    }
+
+    // Ability to use container IAM role for scanning images from Amazon ECR using Scanner deployed as part of Central
+    // Services.
+    CapabilityStatus use_container_iam_role_for_ecr_scanning = 1;
```

Comment:
@dcaravel where did we end up on this for delegated configs?

Reply:
@connorgorman I'm putting some final touches on https://github.com/stackrox/stackrox/pull/6229 which syncs image integrations between central/sensor for local scanning (ETA EoD). Outside of that sync no other changes have been made related to ECR

Reply:
JFYI, I renamed the attribute to `central_scanning_use_container_iam_role_for_ecr` to further highlight that these are capabilities of Central.